pkg/kn/flags/podspec_helper.go

```diff
@@ -255,6 +258,27 @@ func UpdateImagePullSecrets(spec *corev1.PodSpec, pullsecrets string) {
 	}
 }
 
+// UpdateContainers updates the containers array with additional ones provided from file or os.Stdin
+func UpdateContainers(spec *corev1.PodSpec, containers string) error {
```

Comment:
Looking at the changes and how to add test, I'd like to move the parsing logic to a separate function and make this one just accept `[]Containers` array.